src/App.tsx

```diff
@@ -1,16 +1,21 @@
+import { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import { NewMovie } from './components/NewMovie';
 import moviesFromServer from './api/movies.json';
+import { Movie } from './types/Movie';
 
 export const App = () => {
+  const [movies, setMovies] = useState(moviesFromServer);
+  const handleOnAdd = (movie: Movie) => setMovies([...movies, movie]);
```

Comment:
```suggestion
  const handleAddMovie = (movie: Movie) => setMovies([...movies, movie]);
```
The prefix `on` is used for the props, you don't need to combine them.
Also, it's better to know what actually you add